tcf_website/static/base/tcf-style.css

```diff
@@ -0,0 +1,112 @@
+/* tCF Style Classes */
+
+/*************COLORS VARS**************/
+
+/* Use these variables in CSS stylesheets */
+:root {
+--main-color: #274F97;        /* Indigo/Dark blue */
+--secondary-color: #476eb5;   /* Light blue */
+--accent-color: #D75626;      /* Orange */
+--dark-color: #545454;        /* Dark grey, for headers/buttons */
+--shadow-color: #a6a6a6;      /* Medium grey, used for shadows */
+--background-color: #EBEBEB;  /* Light grey, used for backgrounds */
+/* Note: Can use Bootstrap "light" color and white color */
+}
+
+/*************DEFAULT STYLES**************/
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```

Comment:
what do you mean, like spacing before the line?

Reply:
Oh I mean font sizes

Reply:
Actually the new standard is to use relative units (REM or EM) instead of pixels - see here: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

I'm inclined to just go with the Bootstrap defaults for now, unless we have a significant reason not to.

Reply:
That works
I'll let Shirley know so she can update the style guide

Reply:
Okay so the current sizes correspond to h1, h3, and h5
Which tags do you use? Because semantically it would be better that the sizes correspond to h1, h2, and h3

Reply:
On the landing page I use all of h1, h2, h3, h4, h5, and h6 at least once. May be good to include all of these in the style guide.